seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -270,20 +263,20 @@
     },
     {
       "id": "587d7b88367417b2b2512b44",
-      "title": "Arrow functions with input parameters",
+      "title": "Write Arrow Functions with Parameters",
       "description": [
-        "Just like a normal function, you can pass input parameters into arrow functions.",
-        "<code>const doubler = (item) => item * 2; // doubles input value and returns it</code>",
-        "You can pass more than one parameter as well",
-        "Instructions.",
-        "Write the myConcat function that appends content of the second array, passed as argument arr2 to the first array, passed as arr1.",
+        "Just like a normal function, you can pass arguments into arrow functions.",
+        "<blockquote>// doubles input value and returns it<br>const doubler = (item) => item * 2;</blockquote>",
+        "You can pass more than one argument into arrow functions as well.",
+        "<h4>Instructions</h4>",
```

Comment:
Not related to your PR but since you are at it, Can you please replace this with a `<hr>` instead?